Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM debian:jessie
+
+RUN echo "Upgrading distro..." && \
+    apt-get update > /dev/null && \
+    apt-get upgrade -y > /dev/null
+
+RUN echo "Installing dependencies..." && \
+    apt-get install -y git curl libcurl4-gnutls-dev sqlite3 libsqlite3-dev > /dev/null && \
+    apt-get install -y libtar libtar-dev zip libzip-dev unzip > /dev/null && \
+    apt-get install -y python python-distribute python-virtualenv python-setuptools > /dev/null && \
+    easy_install pip && pip install psycopg2 pymysql
+
+RUN echo "Cleaning up..." && \
```

Comment:
Oh I didn't know that about the file deletion, thanks for the recommendations, I applied your review changes and now have a single `RUN` statement. Also added `--no-cache-dir` to the `pip` commands and changed to `python:2.7-slim` (the one you suggested takes 700 MB)